### DISCUSSION TITLE
Username doesn't update when i change my computer username.

### DISCUSSION THREAD

**Discussion by yrifl** [JanDeDobbeleer/oh-my-posh#2742] [Troubleshoot] [answered]
Can you help me?

I tried to change my current computer username to something new, but oh-my-posh didn't update the username my theme is using, the other themes username is not updating either.  Tried to restart: nothing happens still not updating

My new computer username.
![Screenshot 2022-09-03 132253](https://user-images.githubusercontent.com/79893882/188258654-44517522-9507-425f-bbb4-0264f4fe0411.png)

Current theme im using, Ys.
![Screenshot 2022-09-03 132227](https://user-images.githubusercontent.com/79893882/188258652-51924755-3ee3-40b4-95bf-c8de15732cf7.png)

Im a noob. Please help, thank you

**Comment by lewis-yeung** [comment, marked as answer]
@Cyber2f08 What's the output of `$env:USERNAME` in your PowerShell?

**Reply by JanDeDobbeleer** [reply]
@Cyber2f08 If you want to change the windows USERNAME environment variable string, you can do this in the Advanced User Accounts Control Panel.

Open it by running netplwiz ("C:\Windows\System32\Netplwiz.exe"). Then select the name you want to change and click on the Properties button. Now you get the properties of the selected user account: you can change the User name (, Full name & Description) here. You'll (just) need administrator rights to do this.

**Comment by yrifl** [comment]
Wait its the current username. Do i have to rename the current environment variable value in powershell?

**Reply by yrifl** [reply]
the current one: faiq

**Comment by yrifl** [comment]
omg it's working! Thanks!

**Reply by yrifl** [reply]
i just changed the environment variable value.
